.gitbook/assets/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '2'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+    ports:
+      - 22181:2181
+  
+  kafka:
+    image: confluentinc/cp-kafka:latest
+    depends_on:
+      - zookeeper
+    ports:
+      - 29092:29092
+    environment:
+      KAFKA_BROKER_ID: 1
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092,PLAINTEXT_HOST://localhost:29092
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+
+  telemetry-service:
+    image: coolbung/telemetry-service:release-3.3.0
```

Comment:
@coolbung - Is the reference correct? I assume it should be pointing to sunbird image?

Reply:
Couldn't find an image for release-3.3.0, so have left it to my account.
https://hub.docker.com/r/sunbird/telemetry-service/tags (No release-3.3.0)
https://hub.docker.com/r/sunbirddev/telemetry-service/tags (This has RC1, but not release-3.3.0)

The previous step includes creation of the image, so have added a suggestion in the docs to change the telemetry image path in the installation guide.

Reply:
@keshavprasadms - Can you please point to the right image for telemetry service?

Reply:
@coolbung Have pushed the image. Please use [release-3.3.0_RC1](https://hub.docker.com/layers/sunbird/telemetry-service/release-3.3.0_RC1/images/sha256-01bea0aa1643f9f9130b432945e40330b26c70f7066efb7fc0bd40b1e01f1c80?context=explore)

Reply:
Thanks. Have updated the docker compose file to use this. Please merge if all is good

Reply:
Hi @SanthoshVasabhaktula can you help merge this